### PR TITLE
Feature / TEC-3654 improved ui single event tab

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Updates the Customizer UI's Single Event tab to include headings and other copy changes

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3654

**Screenshots:**

Screen Shot 2020-11-25 at 1 59 01 PM

### CODE REVIEW

File: src/Tribe/Customizer/Single_Event.php
Comment (tr1b0t) [inline src/Tribe/Customizer/Single_Event.php:106]: :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 3525d3650b1d812bfd7ced4e308f238b</sub>

File: src/Tribe/Customizer/Single_Event.php
Comment (tr1b0t) [inline src/Tribe/Customizer/Single_Event.php:137]: :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: ad75d64a424c46efc5f4e30645aab3a6</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 4 error(s)

**Review by paulmskim — APPROVED** [review]

**Comment by paulmskim** [comment]
Let's wait until common is merged, but this looks good

**Pull request merged**
